packages/webapp/src/_app/ui/pie-chart.tsx

```diff
@@ -0,0 +1,57 @@
+interface Props {
+    percent: number;
+    size: number;
+    backgroundColor?: string;
+    fillColor?: string;
+}
+
+export const PieChart = ({
```

Comment:
I'm being a bit picky, but this isn't a pie *chart* (or at least isn't much of one). I'd like to name it a bit more accurately. Either call it a pie-status-indicator or pie-countdown-clock. or a countdown pie-mer. lol. no idea, but a name that wouldn't conjure pictures of multiple differently-colored segments

Reply:
Yeah, I hear you. This component is not the countdown and has nothing to do with time, but it's not a full-fledged pie chart either.